src/NHibernate.Test/Generatedkeys/Identity/MyEntityIdentity.cs

```diff
@@ -2,7 +2,10 @@ namespace NHibernate.Test.Generatedkeys.Identity
 {
 	public class MyEntityIdentity
 	{
+		// Assigned by reflection
+#pragma warning disable CS0649 // Field is never assigned to, and will always have its default value
 		private int id;
+#pragma warning restore CS0649 // Field is never assigned to, and will always have its default value
```

Comment:
Likely most of those cases could have been fixed by using a setter instead. But this would require analyzing each test case for ensuring their current way of mapping the property was not having any bound with the tested feature.